src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/InstanceWorkflow.kt

```diff
@@ -0,0 +1,54 @@
+package com.cognifide.gradle.aem.instance.tasks
+
+import com.cognifide.gradle.aem.common.instance.names
+import com.cognifide.gradle.aem.common.tasks.Instance
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.TaskAction
+
+open class InstanceWorkflow : Instance() {
+
+    private val notifier = common.notifier
+
+    @get:Internal
+    val workflow by lazy { common.prop.string("instance.workflow") ?: error("No workflow defined!") }
```

Comment:
In build scripts I used to use error() function but here in plugin impl I prefer to throw typed exception

Reply:
wonder if InstanceException would be ok?